src/components/Configuration/Configuration.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import { Details } from "./components";
+
+const components = {
+  link: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  mode: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  entry: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  filename: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  publicPath: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  advancedOutput: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  expert: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  advancedModule: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  alias: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  advancedResolve: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  hints: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  devtool: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  target: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  externals: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  stats: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  advanced: (children, props) => {
+    return <Details children={children} {...props} />;
+  },
+  libraryTarget: (children, props) => {
+    return <Details children={children} {...props} />;
+  }
+};
+
+const Pre = props => {
+  const newChildren = React.Children.map(props.children.props.children, child => {
+    if (React.isValidElement(child)) {
+      if (child.props.props.className.includes("keyword")) {
+        if (!components[child.props.props.componentname]) return child;
```

Comment:
```suggestion
        if (!detailComponentsList.includes[child.props.props.componentname]) return child;
```

Reply:
This doesn't work, I lost 5 minutes debugging it, JS just run `obj.includes["key"]` just returns `undefined` :D

Reply:
lol :DDDD my bad was editing in github